docs/products/kafka/howto/integrate-service-logs-into-kafka-topic.rst

```diff
@@ -0,0 +1,41 @@
+Integration of logs into Apache Kafka topic
+============================================
+
+You can send logs from your Aiven services into a specified Apache Kafka topic. The setup can be done through `Aiven console <https://console.aiven.io>`_.
+
+.. note::
+
+    Note, the integration can be used for both Aiven for Apache Kafka, as well as external Kafka clusters registered in the project's service integrations page. Read more on :doc:`how to manage Aiven internal and external integrations </docs/tools/cli/service/integration>`.
+
+In this example we'll show how to send the logs from an Aiven for PostgreSQL service to a topic in your Aiven for Apache
+Kafka service. For this example you need:
+
+-  a running Aiven for PostgreSQL service (we'll refer to it as *a source*)
+-  a running Aiven for Apache Kafka service (we'll refer as *a destination*)
+
+Set up Apache Kafka to receive the logs
+----------------------------------------
+
+Verify that Kafka REST API is enabled and create a Kafka topic where you want to receive the logs.
+
+Add a new integration to the source service
+-------------------------------------------
+
+1. Navigate to the Aiven console overview page of the source PostgreSQL service.
+2. Scroll the list of actions till you see **Service integrations**.
+3. Click on **Manage Integrations**. A new pop up window will be displayed showing a list of available integrations for your service.
+4. Select **Kafka Logs** from this list.
+5. Select the destination Kafka service (or external Kafka integration) and the topic where the logs should be pushed.
+
+Test the integration
+--------------------
+
+1. Go to the destination service overview page.
```

Comment:
Should we state that these steps can be done only if your target Kafka is within Aiven? (and if it's an external kafka just check that the topic receives data)